packages/runtime/client/rest/src/index.ts

```diff
@@ -102,7 +107,15 @@ export interface RestAPI {
   ): Promise<Transactions.GetTransactionsForContractResponse>;
   //   createTransactionForContract: Transactions.POST; // - https://docs.marlowe.iohk.io/api/create-transactions // TODO: Jamie, lets unify names
   //   getTransactionById: Transaction.GET; // - https://docs.marlowe.iohk.io/api/get-transaction-by-id
-  //   submitTransaction: Transaction.PUT; // - Jamie is it this one? https://docs.marlowe.iohk.io/api/create-transaction-by-id? If so, lets unify
+  /**
+   * Submit a signed transaction that applies inputs to a contract.
```

Comment:
not a requirement, but you can cross link between API calls so that people know that they submit what was created with X.

For example look at [this endpoint](https://github.com/input-output-hk/marlowe-ts-sdk/blob/main/packages/runtime/client/rest/src/index.ts#L66)

I need to solve the TODO there 😅

Reply:
You mean to give hints in the documentation of how things fit together? If so I think that is a great idea and I will apply it.

Reply:
Exactly, hyperlinks 💪🏻

Reply:
I linked both to the procedure for signing the transaction and the procedure that produces the original unsigned transaction.
I also went ahead and changed the names as they appear in the Rest API documentation. eg
- `submitTransaction` to `submitContractTransaction` according to [Submit contract input application](https://docs.marlowe.iohk.io/api/submit-contract-input-application).
- `createTransactionForContract` to `applyInputsToContract` according to [Apply inputs to contract](https://docs.marlowe.iohk.io/api/apply-inputs-to-contract).